planet-learning/catascript/README.md

```diff
@@ -27,20 +27,24 @@ In order to be able to launch catascript as part of a processed launched by `doc
 This means, if taking into consideration the structure given in `docker-compose.yml`, we need to have : 
 
 ```
-DATA_ROOT = /planet-learning/data
-PATH_TO_EXTRACTED_TICS = /planet-learning/data/processed/dict_TIC.pickle
-PATH_TO_CONFIRMED_CATALOG = /planet-learning/data/confirmed/confirmed_catalog.csv
+DATA_ROOT=/planet-learning/data
+PROCESSED_DIR=processed
+EXTRACTED_TICS_FILE=dict_TIC.pickle
+CONFIRMED_DIR=name_of_the_folder_containing_the_confirmed_planets #confirmed
+CONFIRMED_CATALOG_FILE=name_of_the_file_holding_the_confirmed_planets #transit_confirmed_planets_2019.05.06_09.47.23.csv
 ```
+
 #### Other environment variables
 
 This is an example of values for the other environment variables : 
+
 ```
-LIST_DB_FIELDS = ID,version,HIP,TYC,UCAC,TWOMASS,SDSS,ALLWISE,GAIA,APASS,KIC,objType,typeSrc,ra,dec
-OTHER_MISSIONS_IDS = HIP,TYC,UCAC,TWOMASS,SDSS,ALLWISE,GAIA,APASS,KIC
```

Comment:
It would be a good idea to specify explicitly that no spacing must be around the "=" sign.